### PR TITLE
unparticicpate

### PR DESCRIPTION
**Pull request by somniator18**
Que c'est beau !

### CODE REVIEW

File: app/controllers/complaints_controller.rb [hunk 1]
```diff
@@ -100,6 +100,19 @@ def newParticipation
     end
   end
 
+  # GET /complaints/unparticipate
+  def unparticipate
+    @participation = Participation.getParticipationByUserAndComplaint(current_user, @complaint)
```
Comment (ranska) [inline app/controllers/complaints_controller.rb]: no caml case in ruby method please

**Comment by RynnHeldeD** [comment]
Somniator.tu_sais_quoi_faire();

**Comment by michel-p** [comment]
Roses are red, Caml is good, Ruby's bad, i'm ashamed... it's true !

File: app/models/participation.rb [hunk 2]
```diff
@@ -1,4 +1,11 @@
 class Participation < ActiveRecord::Base
   belongs_to :user
   belongs_to :complaint
+
+  def self.getParticipationByUserAndComplaint(user, complaint)
+    return Participation.where(
+        "user_id = :user_id AND complaint_id = :complaint_id",
+        { user_id: user.id, complaint_id: complaint.id }
```
Comment (ranska) [inline app/models/participation.rb]: not the best syntax

**Pull request merged**